user/deployment/s3.md

```diff
@@ -113,6 +113,9 @@ This bucket policy grants the public read permission:
 }
 ```
 
+Also, if your bucket enables [S3 Block Public Access](https://aws.amazon.com/jp/blogs/aws/amazon-s3-block-public-access-another-layer-of-protection-for-your-accounts-and-buckets/),
```

Comment:
The URL appears to point to a `jp` doc. The document itself is written in English, but links from this page _may_ (unconfirmed) not be. It is probably best to point to what I believe is the equivalent: https://aws.amazon.com/blogs/aws/amazon-s3-block-public-access-another-layer-of-protection-for-your-accounts-and-buckets/